README.md

```diff
@@ -1,2 +1,89 @@
 # github-release-orb
-Civis Analytics' CirleCI Orb repo.
+Civis Analytics' CircleCI Orb repo.
+
+## Usage
+
+Example Config:
+
+```
+version: 2.1
+
+orbs:
+  github-release: civisanalytics/github-release@x.y.z
+
+jobs:
+  release:
+    executor: github-release/github
+  steps:
+    - github-release/<command>
+```
+
+## Build
+In order to build the orb you can run the following command in the root directory of this repository.
+
+`circleci config pack src > orb.yml`
+
+Do not commit the created `orb.yml` file. This file is for publishing to CircleCI when finished.
+
+## Required Environment Variables
+In order for this orb to work, you must have three Environment Variables set:
+1. `GITHUB_TOKEN` - Your GitHub personal access token.
+2. `CIRCLE_PROJECT_USERNAME` - GitHub repository's user or organization. This should be set automatically.
+3. `CIRCLE_PROJECT_REPONAME` - GitHub repository name. This should be set automatically.
+
+## Commands
+
+### Publish
+Publish a new GitHub Release if the last version does not match the current version. This command is useful for running within a custom job. If no custom job is needed, we recommend using the built-in `publish-github-release` job provided by this orb.
+
+| Parameter | Type | Description | Required | Default |
+| --- | --- | --- | --- | --- |
+| `current-version` | string | Current tagged version of the new release | true | |
```

Comment:
I'm a little confused by this variable/description. Should the variable just be `version`? Based on the rest of the code, it seems like this variable describes the most recently released version. That it describes the current version when not changing but the new version when changing. Would an appropriate description be `Version of this codebase. Will create a new github release if different from the previous version.`?

Reply:
Ya that makes sense